active/NumberBox/NumberBox.md

```diff
@@ -118,28 +95,22 @@ with a "///" comment above the member or type. -->
 | StepFrequency | Gets or sets the value part of a value range that steps should be created for. |
 
 ## API Details
-<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+<!-- todo: Missing some APIs in here, like TemplatSettings class and event args classes -->
 
 ```c++ 
 enum NumberBoxSpinButtonPlacementMode
 {
     Hidden,
+    Compact,
     Inline,
 };
 
-enum NumberBoxBasicValidationMode
+enum NumberBoxValidationMode
 {
     InvalidInputOverwritten,
-    IconMessage,
-    TextBlockMessage,
     Disabled,
 };
 
-runtimeclass NumberBoxValueChangingEventArgs
-{
-    Boolean IsContentChanging;
-};
-
 runtimeclass NumberBoxValueChangedEventArgs
 {
     System.Windows.Controls.UndoAction Action;
```

Comment:
The current event args look like this:
```
runtimeclass NumberBoxValueChangedEventArgs
{
    Double OldValue{ get; };
    Double NewValue{ get; };
};
```

Reply:
The [original event args](https://docs.microsoft.com/en-us/dotnet/api/system.windows.controls.textchangedeventargs.-ctor?view=netframework-4.8#System_Windows_Controls_TextChangedEventArgs__ctor_System_Windows_RoutedEvent_System_Windows_Controls_UndoAction_) were chosen to maintain alignment with TextBox's events. But seeing as how we are already diverging, the args you implemented are certainly more clear + easy to use and makes the most sense for the scenarios this control is used in. 

I have the spec updated to match. 😊